recipes/libftdi/1.x/conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibFtdiConan(ConanFile):
+    name = "libftdi"
+    description = "A library to talk to FTDI chips"
+    license = "LGPL-2.0-only", "GPLv2-or-later"
+    topics = ("conan", "libftdi1")
+    homepage = "https://www.intra2net.com/en/developer/libftdi/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package", "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+            "shared": [True, False], 
+            "fPIC": [True, False]
+    }
+    default_options = {
+            "shared": False, 
+            "fPIC": True
+    }
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "libftdi1-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("Building with Visual Studio is not supported")
+
+    def build_requirements(self):
+        if not tools.which("pkg-config"):
+            self.build_requires("pkgconf/1.7.3")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        options = {
+            "BUILD_TESTS": False,
+            "EXAMPLES": False,
+            "FTDI_EEPROM": False,
+            "FTDIPP" : True,
+            "STATICLIBS": not self.options.shared
+        }
+        self._cmake.definitions.update(options)
+        self._cmake.configure()
+        return self._cmake
+
+    def requirements(self):
```

Comment:
Does this need libconfuse?

Reply:
libconfuse is only required by the eeprom utility, not by the library.